src/parcsr_mv/par_csr_matmat.c

```diff
@@ -0,0 +1,159 @@
+/******************************************************************************
```

Comment:
Maybe these should be put together with hypre_ParCSRMatMat<Host, Device> codes? Not sure how much additional work that will be to move those into this file or these into par_csr_triplemat.c.

Reply:
My idea is to move the `ParCSRMatMat` codes to this new file actually, since `triplemat` became quite big